certified-connectors/Talkdesk/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "YourTalkdeskClientId",
+          "scopes": [
+            "contacts:read agents-bulk:write callback:write notifications:write industries-activity:write users:read presence-user:read account-custom-status:read ccaas-user-status:write webhooks-trigger:write webhooks-schema:read account:read presence-account:read"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://YourTalkdeskAccount.talkdeskid.com/oauth/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://YourTalkdeskAccount.talkdeskid.com/oauth/token"
+            },
+            "refreshUrl": {
+              "value": "https://YourTalkdeskAccount.talkdeskid.com/oauth/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#bf91fc",
+    "capabilities": [],
+    "publisher": "Talkdesk",
+	"stackOwner": "Talkdesk"
+  }
+}
```

Comment:
@StefanCRT22 The icon is not required to open source, please remove it from the PR.